### ISSUE TITLE
freetype.Font.render should also support newlines

### ISSUE THREAD

**Issue by denilsonsa** [pygame-community/pygame-ce#2735]
I see that [pygame.font.Font.render](https://pyga.me/docs/ref/font.html#pygame.font.Font.render) supports newlines since pygame-ce-2.1.4.

However, [pygame.freetype.Font.render](https://pyga.me/docs/ref/freetype.html#pygame.freetype.Font.render) doesn't seem to support it. (I haven't tested, I'm assuming based on the documentation.) I believe the FreeType module should also be updated to have roughly the same features as the Font module.

**Comment by denilsonsa** [comment]
But with the FreeType module I can get extra information about the font
(such as family, and subfamily variant), while that info isn't available on
the Font module.

(Please note I have limited connectivity these days and I can't
double-check what I'm saying. I just wanted to give you a response as
quickly as possible. Please forgive me for mistakes.)

**Comment by MyreMylar** [comment]
Yes, we are trying to add the missing features you mention to font rather
than the other way around.

**Comment by Starbuck5** [comment]
Hello, just combing through old issues.

> But with the FreeType module I can get extra information about the font
(such as family, and subfamily variant), while that info isn't available on
the Font module.

Between freetype.Font and font.Font I see both support `.name` and `.style_name`, which are the only things I can find to return information like that.

I'm going to close this is not planned because we want to replace freetype.Font, so it's not worthwhile to add newline support to it at this time.
